webcompat/templates/home-page/form.html

```diff
@@ -15,9 +15,25 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
       <div class="r-Grid r-Grid--withGutter">
         <div class="r-Grid-cell r-minM--1of2">
           <div class="wc-Form-group js-Form-group">
+            <div class="wc-Form-information">
+              {{ form.url.label(class_='wc-Form-label') }}
+              {% if form.url.errors %}
+                {% for error in form.url.errors %}
+                  <span class="wc-Form-helpMessage">{{ error }}</span>
+                {% endfor %}
+              {% endif %}
+            </div>
+            <span class="wc-Form-validation">
+            {{ form.url(class_='wc-Form-item required',
+               placeholder='e.g. https://example.com',
+               required=True, type='url', pattern='') }}
+            </span>
+          </div>
+
+          <div class="wc-Form-group js-Form-group">
             <fieldset class="{% if form.problem_category.errors %}is-error{% endif %}">
               <div class="wc-Form-information">
-                <label class="wc-Form-label">What seems to be the trouble? <span class="wc-Form-required">*</span></label>
+                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

Comment:
@karlcow in the latest pr version, I made all requested changes except this one. I wanted to get Mike's reasoning why the original code used label for radio buttons differently. if too much time passes without any response, I will change to your suggestion though.